### DISCUSSION TITLE
q-dialog set custom position

### DISCUSSION THREAD

**Discussion by bagladivyang03** [quasarframework/quasar#9446] [CLI - PWA mode]
Hello All,
Actually, I was trying to find a way to set the custom position for q-dialog but couldn't find any way to do so,
there is just option of left, right, bottom, top, standard.

 So can anyone pls help how to do this, It would be a great help.

**Comment by ontwikkelfabriek** [comment]
@bagladivyang03 

Here's a codepen where I use margins to position the dialog:
https://codepen.io/ontwikkelfabriek/pen/mdWBmbe

**Reply by bagladivyang03** [reply]
Thanks @ontwikkelfabriek !!!
